pkg/proc/proc_windows.go

```diff
@@ -663,6 +663,20 @@ func (dbp *Process) resume() error {
 	return nil
 }
 
+func (dbp *Process) detach() error {
```

Comment:
If you are introducing detach function, I would get rid of ptrace_windows.go file altogether. PtraceAttach and PtraceDetach function names do not make sense on windows. I was confused when reading code. Maybe not in this PR.